### ISSUE TITLE
[feature request] Debugger break on unlogged opcode execution

### ISSUE THREAD

**Issue by nesrocks** [TASVideos/fceux#379]
Request: Add a toggle for breaking execution when encountering an unlogged opcode (as in the code/data logger logs).
Why: When reverse engineering games this would be very useful for detecting new branches in code. Play around in a section of the game with CD on, then turn on the "break on unlogged opcode" toggle, and then execute the new action.

**Comment by mjbudd77** [comment]
Would you want this to break on unclogged code and data separately? One selection for break on unclogged code and one for break on unclogged data?

**Comment by nesrocks** [comment]
Having a sepparate option for "break on unlogged code" and "break on unlogged byte read" would be very useful.
By the way, it is fine to have this on a dropdown menu in the debugger, it doesn't have to be directly accessible on the debugger UI which already has too many items.

**Comment by mjbudd77** [comment]
I've added the code for this to the Qt GUI. In addition, I have fixed a couple items you had mentioned in the nes dev thread. Namely, I added the cd display characters to the debugger assembly view and added code for most of the debug and tools windows to remember their positioning. Please try out the qfceux.exe from the appveyor build at the commit seen above.

**Comment by warmCabin** [comment]
Added to win32 as well.

**Comment by mjbudd77** [comment]
@nesrocks can you please provide feedback? Please do your testing on the Qt GUI, I don’t think the changes have actually made it to the Win32 master branch

**Comment by warmCabin** [comment]
That's right, Win32 development is [still ongoing](https://github.com/TASVideos/fceux/pull/368). You should be able to grab the [latest build](https://ci.appveyor.com/project/zeromus/fceux/builds/40334718) from AppVeyor if you want to test that version. I added a dropdown for those `break_on_xxx` flags you made, mjbudd.

**Comment by nesrocks** [comment]
> @nesrocks can you please provide feedback? Please do your testing on the Qt GUI, I don’t think the changes have actually made it to the Win32 master branch

I'm sorry just after making this request I got word from the publisher that my game that I had been working on for 4 years was about to be released on consoles, which means a lot of work that needs to be done fast. I did test it though and it did seem to have a few issues. Just never had the time to test more so I could write down a consistent list of issues. I hope to have enough time to do this soon.
